configure.ac

```diff
@@ -102,6 +102,11 @@ case "$host" in
     ;;
 esac
 
+dnl - platform, for use with game console ports
+AC_ARG_WITH([platform],
```

Comment:
It looks like this just duplicates what the `--host` argument does:

https://www.gnu.org/software/autoconf/manual/autoconf-2.65/html_node/Specifying-Target-Triplets.html

I expect devkitpro probably defines a triplet for the Switch that you can use (although I don't know a lot about it)

Reply:
We don't build Switch specific tools, we're using a standard aarch64-none-elf triplet & using the same tools for a few baremetal (newlib based) targets we're playing with. Adding this option would let us build, for instance, chocolate doom for bare metal Rpi3 if anyone felt like it ;o)